style.css

```diff
@@ -558,3 +473,346 @@ input[type="submit"]:disabled {
+    margin-bottom: 5px;
+  }
+
+  .title {
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .para p {
+    width: 666px;
+    height: 123px;
+    font-size: 20px;
+    line-height: 32px;
+    margin-bottom: 20px;
+  }
+
+  .whole {
+    display: flex;
+    gap: 100px;
+  }
+
+  .icons {
+    margin-left: 0;
+    display: flex;
+    flex-direction: column;
+    margin-top: 60px;
+  }
+
+  .fas {
+    font-size: 20px;
+  }
+
+  .nxt {
+    display: none;
+  }
+
+  .not {
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 30%;
+  }
+
+  .work {
+    padding: 160px;
+  }
+
+  .work-head {
+    border: none;
+  }
+
+  .group {
+    display: flex;
+    gap: 15px;
+  }
+
+  .imge img {
+    margin-top: 35px;
+  }
+
+  .imge-txt {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .work1-img {
+    width: 684px;
+    height: 448px;
+  }
+
+  .work-head2 {
+    font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .work1-para {
+    width: 448px;
+    height: 140px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .work1-list li {
+    padding: 10px 12px;
+    font-size: 15px;
+  }
+
+  .work-menu {
+    margin-bottom: 30px;
+  }
+
+  .p {
+    margin-left: -40px;
+  }
+
+  .btn {
+    padding: 12px;
+    font-size: 17px;
+    line-height: 24px;
+    margin-top: 60px;
+  }
+
+  .gridy {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto auto;
+    gap: 15px;
+  }
+
+  .work-title2a {
+    background-image: linear-gradient(180.45deg, rgb(38 38 38 / 0%) 0.75%, rgb(38 38 38 / 90%) 84.18%), url("img/Img\ \(1\).png");
+  }
+
+  .go {
+    visibility: hidden;
+  }
+
+  .be {
+    background-image: linear-gradient(179.35deg, rgb(38 38 38 / 0%) 0.85%, rgb(38 38 38 / 90%) 84%), url("img/Img\ \(2\).png");
+  }
+
+  .bee {
+    background-image: linear-gradient(179.4deg, rgb(38 38 38 / 0%) 0.81%, rgb(38 38 38 / 90%) 84.03%), url("img/Img\ \(3\).png");
+  }
+
+  .be a,
+  .bee a {
+    display: none;
+  }
+
+  .bee h1 {
+    margin-top: 20px;
+  }
+
+  .work-he {
+    color: white;
+    line-height: 44px;
+    font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
+    font-weight: 400;
+    font-size: 32px;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .work-head22 {
+    visibility: hidden;
+  }
+
+  .work2-para {
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .bet li {
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .about {
+    padding: 160px;
+    padding-right: 0;
+  }
+
+  .about-title {
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  .about-para {
+    width: 566px;
+    height: 148px;
+    font-size: 20px;
+    line-height: 32px;
+  }
```

Comment:
[OPTIONAL] Hello @Uchenna-ibute, I'm not sure why no other reviewer told you about this. I've noticed that you used a fixed width for your whole project. Please take into account that, although this is the width displayed on the design, you shouldn't use a fixed width for your projects (it is just the width of the device used for the screenshot). I will explain: 

The idea of Responsive Web Design is that your webpage should adapt itself to any device/window size that opens it. In other words, it is responsive. This means that using a fixed width for your sections will break this responsiveness, causing a static view to be formed. So, what can you do about it? Simple, use percentages! Instead of `width: 327px`, use `width: 100%`. This will make your project stretches to any window size that you can imagine 😉. If you need a smaller section, use a lower percentage. If you really need to use a fixed width, keep in mind that it may break the responsiveness of your webpage 😄! 

_But, my mobile version looks horrible stretched to a big window 😢._ 

Don't worry about this! This is why you are using [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries), allowing different styling for different window sizes. So, you can edit a mobile version and a desktop version for your apps, depending on the device 😄.

With a fixed width it doesn't fit correctly. Check your project on larger windows: 
![image](https://user-images.githubusercontent.com/42799579/206632745-7f2c773c-3485-4d17-941e-1702d82bd32a.png)

If you are interested to learn more about it, check [this link](https://www.w3schools.com/html/html_responsive.asp) and this [youtube video](https://www.youtube.com/watch?v=VQraviuwbzU). 

Good luck and keep coding 😃!